src/lib/y2partitioner/widgets/device_menu_button.rb

```diff
@@ -129,6 +129,7 @@ def widget_id_for_action(action)
       # @param id [Symbol, String]
       # @return [Hash]
       def action_for_widget_id(id)
+        return nil if id.nil?
```

Comment:
should be nil or exception? if so, then please document it that it can also return nil.

Reply:
Actually, I would never expect `#handle` to be called with no arguments (or with a nil argument) since `handle_all_events` is used.

Unfortunately, supporting that case is needed in order to be able to use `include_examples "CWM::AbstractWidget"` which always calls `#handle` with no arguments, so I have to adapt the class to ignore such wrong calls.

Reply:
ah, yeah. That is problematic, because for calls it uses arity and rspec mock basically mocks call with *args, so -1 arity and it breaks this test. Please add comment for it, as it is quite tricky.